src/components/Auth/useAuthProvider.ts

```diff
@@ -76,20 +99,19 @@ export const useAuthProvider = () => {
     clear()
   }, [])
 
```

Comment:
@selankon can you please elaborate on why you removed the refresh function?

Reply:
Is not needed enymore. Refresh function was used before i know how that if I want to get the signer address i can just do

```
signer.address()
```

and all the reactive ui will respond to the new address. 

Refresh is not used anymore:

[src/router/Router.tsx](https://github.com/vocdoni/ui-scaffold/pull/769/files/6f0bce579f389f90fa85653a042d30b5ea8352c6#diff-12e851622e898e308434252678d097fb493488174f6fcf90bfff9fe09e7cdad6)